### ISSUE TITLE
Import lodash granulary

### ISSUE THREAD

**Issue by klimashkin** [beaugunderson/ip-address#89]
Hi! I noticed that you've started importing the whole lodash.
https://github.com/beaugunderson/ip-address/blob/master/lib/ipv4.js#L6

That almost doubles our lodash size:
<img width="266" alt="Screen Shot 2019-08-01 at 1 31 54 PM" src="https://user-images.githubusercontent.com/576077/62327851-74c14d80-b466-11e9-9a1a-e7f57894f9c5.png">

All other dependencies import only needed lodash functions like `import map from 'lodash/map';` and we also using `babel-plugin-lodash` to keep only methods that are really used.
But babel is usually not run on node_modules, so import in `ip-address` remains untouched and adds the whole lodash bundle separately in the result.

Could you also use `import X from 'lodash/X' notation? It would help us a lot, thank you!

**Comment by beaugunderson** [comment]
i switched on advice from the creator of lodash because the individual npm
modules were not updated nearly as often as the lodash module itself and
there were ancient security vulnerabilities—I hear you, though, and will
work to find a solution [image: 👍]

**Comment by klimashkin** [comment]
I tried to use `babel-plugin-lodash` directly on `ip-address`:
```js
{
  test: /\.js$/,
  include: [/node_modules\/ip-address/],
  use: {
    loader: 'babel-loader',
    options: {
      plugins: ['lodash'],
    },
  },
},
```
But it doesn't work, because `babel-plugin-lodash` works only with esm imports, whilst `ip-address` is distributed only as compiled cjs without a way to import es (for instance by having `es*` entry point in package.json)

**Comment by beaugunderson** [comment]
@klimashkin gotcha--for the record i have nothing against updating ip-address to use newer language features as long as we can keep backwards compatibility via compiling it using e.g. babel, the only reason it has stuck to older language features is because most of the code dates back to 2011! :)

**Comment by klimashkin** [comment]
Seems like was fixed in #92
